2_API_specification/Suggestions/SteffenBrinckmann/README.md

```diff
@@ -0,0 +1,67 @@
+# Light-weight Python-based No-Library API
+This is a "API" that I adopted into after 4 iterations. It is meant as a suggestion and starting point for a possible discussion.
+
+## General
+### Advantages
+- extraction does not require custom libraries
+- can be used in command-lines and dynamically imported into python (see below for code)
+- validators are easy: diff file head and tail and test if given example is extracted
+- (additional ...)
+
+### Disadvantages
+- has some python baggage around it, compared to straight conversion
+- uses an unconventional approach
+- Other languages (R, etc.) required either translation or a wrapper. I have written a CWL wrapper.
+- (additional ...)
+
+
+## Implementation
+### Extractor arguments
+- file path (could be file object to allow for data stream)
+- recipe (allows to customize extraction)
+- save fig (allows to save a "thumbnail" to file)
```

Comment:
To kick of the discussion ;): I feel this does not necessarily need to be a default behavior